services/cedar.go

```diff
@@ -0,0 +1,85 @@
+package services
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/evergreen-ci/aviation"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+type userCredentials struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
+
+// DialCedar is a convenience function for creating a RPC client connection
+// with cedar via gRPC. The username and password are the LDAP credentials for
+// the cedar service.
+func DialCedar(ctx context.Context, client *http.Client, baseAddress, rpcPort, username, password string, retries int) (*grpc.ClientConn, error) {
```

Comment:
can we make a CedarOptions struct with everything in this signature except for the context and the client.